app/views/papers/_form.html.erb

```diff
@@ -53,6 +53,16 @@
     <%= f.text_field :tag_list, value: paper.tag_list.join(",") %>
   </div>
 
+  <div class="field">
+    <%= f.label :methodology_list, 'Methodologies (the following methodologies are accepted, please separate with a comma: meta-analysis, systematic review, randomized control trial, quasi-experiment, cohort, case-control, cross-sectional survey, case report)' %>
```

Comment:
Later when we force auto-complete on the frontend, we can ensure they can't input anything else. Right now this is fine.